### PR TITLE
PR 1 - Implement New toolkit version csharp samples.

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Sample List -** 

**1. tab-personal/mvc-csharp
2. tab-sso/csharp**

### CODE REVIEW

File: samples/tab-sso/csharp/TeamsApp/TeamsApp.ttkproj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" Sdk="Microsoft.TeamsFx.Sdk">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>42a0237c-30e0-42ae-b6ed-50f318517e77</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
```
Comment (Alive-Fish) [inline samples/tab-sso/csharp/TeamsApp/TeamsApp.ttkproj:6]: This `ItemGroup` is unnecessary and can be deleted, but it doesn't block usage.

File: samples/tab-personal/mvc-csharp/TeamsApp/TeamsApp.ttkproj [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" Sdk="Microsoft.TeamsFx.Sdk">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>42a0237c-30e0-42ae-b6ed-50f318517e77</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="appPackage\color.png" />
+    <None Include="appPackage\manifest.json" />
+    <None Include="appPackage\outline.png" />
```
Comment (Alive-Fish) [inline samples/tab-personal/mvc-csharp/TeamsApp/TeamsApp.ttkproj:9]: This `ItemGroup` is unnecessary and can be deleted, but it doesn't block usage.

**Review by Alive-Fish — APPROVED** [review]

File: samples/tab-sso/csharp/TeamsApp/TeamsApp.ttkproj (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-sso/csharp/TeamsApp/TeamsApp.ttkproj:6]: Fixed

File: samples/tab-personal/mvc-csharp/TeamsApp/TeamsApp.ttkproj (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-personal/mvc-csharp/TeamsApp/TeamsApp.ttkproj]: Fixed

**Review by Alive-Fish — APPROVED** [review]

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Mohammed-MSFT — APPROVED** [review]
Looks good, Approved.

**Pull request merged**